update.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Authors:
+#   Dishant Khanna <dishant.khanna1807@gmail.com> [FOSSASIA]
+
+# Description:
+#   An update bash script for engelsystem
+
+# Add upstream 
+git remote add upstream http://www.github.com/fossasia/engelsystem.git
+
+# Pull the updates
+git pull upstream development
```

Comment:
> which includes pulling from my repository
Pulling from your repository? This code

```
git remote add upstream http://www.github.com/fossasia/engelsystem.git
git pull upstream development
```
shows that it will pull from http://www.github.com/fossasia/engelsystem.git, not your repo.